models/commons/src/initiator/generator.ts

```diff
@@ -0,0 +1,16 @@
+import type { TInitiator } from './types';
+import Reference from '../reference';
+import { Generator, fake } from '@commercetools-test-data/core';
+
+const generator = Generator<TInitiator>({
+  fields: {
+    isPlatformClient: fake((f) => f.random.boolean()),
+    externalUserId: fake((f) => f.random.uuid()),
+    anonymousId: fake((f) => f.random.uuid()),
+    clientId: fake((f) => f.random.uuid()),
+    customerRef: Reference.random().build(),
```

Comment:
don't we need to specify the `typeId=customer` here. I mean is the `typeId` optional in references

Reply:
good catch!

Reply:
https://github.com/commercetools/test-data/pull/58/commits/df91c001e90f80c4bc44857993a8befec3197d92